tests/plugins/os/unix/test_trash.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime, timezone
+from io import BytesIO
+
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.plugins.os.unix._os import UnixPlugin
+from dissect.target.plugins.os.unix.trash import GnomeTrashPlugin
+from dissect.target.target import Target
+from tests._utils import absolute_path
+
+
+def test_gnome_trash(target_unix_users: Target, fs_unix: VirtualFilesystem) -> None:
+    """test if GNOME Trash plugin finds all deleted files including recursively deleted folders and expunged items."""
+
+    fs_unix.map_file_fh("etc/hostname", BytesIO(b"hostname"))
+    fs_unix.map_dir("home/user/.local/share/Trash", absolute_path("_data/plugins/os/unix/trash"))
+    target_unix_users.add_plugin(UnixPlugin)
+    target_unix_users.add_plugin(GnomeTrashPlugin)
+
```

Comment:
```suggestion

    assert hasattr(target_unix_users, "recyclebin")

```

Reply:
Forgot about that one, even though I recently used it. Feel free to change it to `target_unix_users.has_function("recyclebin")`